ca/server.go

```diff
@@ -69,6 +69,30 @@ func (s *Server) SetReconciliationRetryInterval(reconciliationRetryInterval time
 	s.reconciliationRetryInterval = reconciliationRetryInterval
 }
 
+// GetUnlockKey is responsible for returning the current unlock key used for encrypting TLS private keys and
+// other at rest data.  Access to this RPC call should only be allowed via mutual TLS from managers.
+func (s *Server) GetUnlockKey(ctx context.Context, request *api.GetUnlockKeyRequest) (*api.GetUnlockKeyResponse, error) {
+	log.G(ctx).WithFields(logrus.Fields{
+		"method": "GetUnlockKey",
+	})
+
+	resp := api.GetUnlockKeyResponse{}
+	s.store.View(func(tx store.ReadTx) {
+		cluster := store.GetCluster(tx, s.securityConfig.ClientTLSCreds.Organization())
```

Comment:
Do not use the store here. Change `(*Server).updateCluster` to store the current KEK in the `Server` struct and use that here (protected by `s.mu`).